pkg/client/clientset/versioned/fake/register.go

```diff
@@ -50,7 +48,6 @@ func init() {
 // After this, RawExtensions in Kubernetes types will serialize kube-aggregator types
 // correctly.
 func AddToScheme(scheme *runtime.Scheme) {
-	brokerv2alpha1.AddToScheme(scheme)
-	brokerv2alpha2.AddToScheme(scheme)
-	brokerv2alpha3.AddToScheme(scheme)
```

Comment:
why do you remove those lines?

Reply:
It should be the result of my running code-gen tool. Not sure if it's used at all. I didn't have problems when I do testing.
if you found they are used I can add them back.